src/features/сart/CartSlice.ts

```diff
@@ -0,0 +1,90 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { Product } from '@/types/Phone';
+
+interface CartItem extends Product {
```

Comment:
Is this type definitely not going to be used anywhere else? It would be better to move it to a separate file